src/components/overview/Overview.vue

```diff
@@ -145,6 +142,18 @@ export default {
 			}
 			this.loading = false
 		},
+		sortCards(when) {
+			const cards = this.assignedCardsDashboard[when]
+
+			if (!cards) return null
```

Comment:
please make this a block https://docs.nextcloud.com/server/latest/developer_manual/getting_started/codingguidelines.html#control-structures

Reply:
Thank you for making me aware of this; it has been fixed in the second commit :)